### ISSUE TITLE
Use Available Cellphones Data connection over Wi-Fi as WAN 

### ISSUE THREAD

**Issue by JRTechServices** [Ysurac/openmptcprouter#2706]
It would be very cool someday to use cellphones that are connected via Wi-Fi to your network as an OMR WAN interface. Imagine if you could tap several cellphones over Wi-Fi and only use a sliver thats devices cellular bandwidth. I thought would share this idea. Has it been done before? I know you can turn on a moble hotspot but is this possible?

**Comment by Ysurac** [comment]
Yes it's possible and working, but not very well: you use a mobile connection, so with packets loss, over another wifi connection that also have packets loss.
